src/routes/news/[slug].svelte

```diff
@@ -44,7 +54,31 @@
                 [BLOCKS.EMBEDDED_ASSET]: (node, next) => {
                     const asset = assetMap.get(node.data.target.sys.id);
                     return `<img src=${asset.url} alt="${asset.title}" />`;
-                }
+                },
+                [BLOCKS.EMBEDDED_ENTRY]: (node, next) => {
+                    // https://www.contentful.com/blog/2021/04/14/rendering-linked-assets-entries-in-contentful/
```

Comment:
This comment provides some value as it refers to relevant documentation. All other comments should be removed. We should strive to keep the code self documented through proper naming conventions.